tests/Jira/ApiTest.php

```diff
@@ -149,6 +149,40 @@ public function testFindVersionByName()
 		);
 	}
 
+	public function testGetResolutions()
+	{
+		$response = '[{"self":"https://test.atlassian.net/rest/api/2/resolution/1","id":"1","description":"A fix for this issue is checked into the tree and tested.","name":"Fixed"},{"self":"https://test.atlassian.net/rest/api/2/resolution/10000","id":"10000","description":"This issue won\'t be actioned.","name":"Won\'t Do"}]';
```

Comment:
Guess thats a preference of style. I'd done it like this to more explicitly model what goes in and out.